libs/features/mas/src/wcs.js

```diff
@@ -296,11 +323,11 @@ export function Wcs({ settings }) {
                 let group = queue.get(groupKey);
                 if (!group) {
                     const options = {
-                        country,
-                        locale,
+                        country: validCountry,
+                        locale: validLocale,
+                        ...((country !== 'GB' && !perpetual) && { language: validLanguage }),
```

Comment:
i don't understand this test has already been done to elaborate validLanguage, hasn't it?